webapp/src/main/resources/public/js/stores/branches/BranchesHistoryStore.js

```diff
@@ -45,6 +46,20 @@ class BranchesHistoryStore {
         this.openBranchStatistic = openBranchStatistic;
     }
 
+    changeCreatedBefore(createdBefore) {
+        this.createdBefore = createdBefore
+            ? moment(createdBefore)
+                .format("YYYY-MM-DDTHH:mm")
```

Comment:
Do we use this construct in the other filter?

Reply:
No, we are using react state to control the filters in the other filter, but are not here. I had to change some settings to get the page behavior to work correctly